src/java/org/apache/cassandra/streaming/StreamWriter.java

```diff
@@ -75,8 +76,13 @@ public StreamWriter(SSTableReader sstable, Collection<Pair<Long, Long>> sections
     public void write(DataOutputStreamPlus output) throws IOException
     {
         long totalSize = totalSize();
-        logger.debug("[Stream #{}] Start streaming file {} to {}, repairedAt = {}, totalSize = {}", session.planId(),
-                     sstable.getFilename(), session.peer, sstable.getSSTableMetadata().repairedAt, totalSize);
+        if (logger.isDebugEnabled())
+            logger.debug("[Stream #{}] Start streaming file {} to {}, repairedAt = {}, totalSize = {}",
+                         SafeArg.of("planId", session.planId()),
+                         SafeArg.of("file", sstable.getFilename()),
```

Comment:
also would like to doublecheck this is safe

Reply:
sstable file names are safe right?

Reply:
I feel like this could include the absolute path, so I'll mark it (and equivalents) as unsafe

Reply:
switched to UnsafeArg